cloud-functions/common/common.go

```diff
@@ -467,3 +472,23 @@ func GetBackendsIps(ctx context.Context, project, zone string, instancesNames []
 	}
 	return
 }
+
+func CreateBucket(ctx context.Context, project, obsName string) (err error) {
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Error().Err(err).Send()
+		return
+	}
+
+	// Creates a Bucket instance.
+	bucket := client.Bucket(obsName)
+
+	// Creates the new bucket.
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
```

Comment:
I guess it is OK if the clusterize will fail if we fail to create bucket, but instead of retry and complicate it, I think I will just remove the timeout .